src/containers/category-filter.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
```

Comment:
According to the pathwright - `category-filter` should be inside `src/component` presentational component.
![image](https://user-images.githubusercontent.com/31897434/76896135-96eecb00-6891-11ea-89b5-5d0862965cf1.png)
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896829/

Note: The presentational component is a component that is not connected to the redux store or state.

Obviously this component is not connecting to the redux store and is not having state either.